pygeo/constraints/DVCon.py

```diff
@@ -2007,6 +2017,11 @@ def addLinearConstraintsShape(
 
         self._checkDVGeo(DVGeoName)
 
+        if childIdx is not None:
+            DVGeo = self.DVGeometries[DVGeoName].children[childIdx]
```

Comment:
It took me a bit of code reading, but I see how this makes sense. We have a single DVcon object which "receives" the parent DVGeo, and we were previously applying the constraint to this object instead of the children (when present) as expected. This means that we now need to add one of these for each child ffd, right? Can you add a short description of what is going on and how to extract the `childIdx` index from DVGeo?

Reply:
Yes, without `childIdx` the constraint is applied on the parent. You only need to add this constraint to each FFD that needs it. `childIdx` is defined by the order in which you add the child FFD to the parent (the first is 0, second is 1, etc.). I will update the docstrings to include this description.

To be clear though, I took this approach directly from [addLeTeConstraints](https://github.com/mdolab/pygeo/blob/7cf8e4e0b827c12cb2f2e79bd28b44bbe9228f15/pygeo/constraints/DVCon.py#L1857-L1860) which already worked with child FFDs.